### PR TITLE
Developer Documentation on workflow

### PR DESCRIPTION
**Pull request by mts299**
# Scope

This is ahead branch for multiple mini branches.  Currently just shows the outline of the new documentation section.

### CODE REVIEW

**Comment by mts299** [comment]
I think this is more of a format check now

File: docs/index.md [hunk 1]
```diff
@@ -34,23 +34,32 @@ The software is maintained by the SuperDARN Data Analysis Working Group (DAWG) o
...
   - Developers Guide
-    - [Copyrights and Licensing](developers_guide/copyright_license.md) 
+    * [Copyrights and Licensing](dev/copyright_license.md) 
+    * [Communications Guidelines](dev/communication.md)
+    * [Adding a New Feature](dev/feature.md)
+    - Workflow
+        * [GitHub Projects](dev/projects.md)
+        * [Issues](dev/issues.md)
+        * [Creating a Branch](dev/branching.md)
+        * [Pull Requests](dev/pull_requests.md)
```
Comment (mts299) [inline docs/index.md:44]:
``` suggestion
        * [Pull Requests](dev/pull_request.md)
```

**Comment by mts299** [comment]
All done!

**Comment by ecbland** [comment]
Testing this PR was just a formatting check, and it looks good to me. Thanks for putting this together @mts299 !!

**Pull request merged**